src/psystem.jl

```diff
@@ -0,0 +1,153 @@
+module PowerSystem
+
+using ..ExaPF: Parse
+using ..ExaPF: Network
+
+using SparseArrays
+
+"""
+    Pf
+
+This structure contains constant parameters that define the topology and
+physics of the power network.
```

Comment:
As far as I understand, `Pf` creates object in the main memory, and not directly onto the GPU. I would maybe add a comment about that here, just to tell the user that some objects are bound to be exported to the GPU after.

Reply:
Added comment